robot/Cumulus/resources/GAUPageObject.py

```diff
@@ -1,8 +1,31 @@
 from cumulusci.robotframework.pageobjects import ListingPage
 from cumulusci.robotframework.pageobjects import pageobject
+from cumulusci.robotframework.pageobjects import BasePage
 from BaseObjects import BaseNPSPPage
 from NPSP import npsp_lex_locators
 
+
+@pageobject("Custom", "GauAllocation")
+class GauAllocationPage(BaseNPSPPage, BasePage):
+
+    def _is_current_page(self):
+        """
+        Waits for the GAU Allocations iframe to load
+        """
+        self.npsp.wait_for_locator('frame','Manage Allocations')
+        self.npsp.choose_frame('Manage Allocations')
+
+    def set_gau_allocation(self,**kwargs):
+        """
+        Allocates the values for the GAU names as specified by the values passed in kwargs
+        """
+        for key, value in kwargs.items():
+            if 'General Accounting Unit' in key:
+                self.npsp.select_search(key, value)
+            else:
+                self.npsp.add_gau_allocation(key, value)
```

Comment:
this keyword should be moved inside this page object, but I already moved in my PR. I will resolve the conflict once u merge urs